server.js

```diff
@@ -6,6 +6,7 @@ const express = require('express'),
 	mongoose = require('mongoose'),
 	routes = require('./routes'),
 	handlers = require('./handlers');
+const { KeyObject } = require('crypto');
```

Comment:
Ignore this line! Somehow this snuck in here. I'll remove this on the next commit